code-of-conduct.md

```diff
@@ -1,4 +1,9 @@
-# Kubernetes Community Code of Conduct
+---
+title: "Kubernetes Community Code of Conduct"
+weight: 1
+description: |
+  Expectations of conduct that govern all members of the community.
+---
```

Comment:
We don't actually need to import this file, we have a copy of what it links to in the contributor site repo:
https://github.com/kubernetes/contributor-site/blob/master/content/en/resources/code-of-conduct.md

When the community section is created, we can move that doc from resources to community 😄